src/uct/api/uct.h

```diff
@@ -437,6 +437,19 @@ enum {
 
 
 /**
+ * @ingroup UCT_RESOURCE
+ * @brief   @anchor UCT_FLUSH_FLAGS Flags that control completion semantic.
+ */
+enum {
+    UCT_FLUSH_FLAG_LOCAL = 0,           /**< Guarantees flush of local
```

Comment:
this change should be part of another pr - "Add active flush to UCT". That PR would be the basis of implementing keepalive mechanism in UCP.